version.py

```diff
@@ -1 +0,0 @@
-VERSION = '1.3.0'
```

Comment:
Similarly, I feel that refactoring could have been it's own PR to improve readability.

Reply:
Ok I get it now.